test-utils/test-framework/src/utils/yagna_cli.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::anyhow;
+
+use crate::YagnaMock;
+
+pub trait YagnaCli {
+    fn appkey_list_json(&self) -> anyhow::Result<Vec<serde_json::Value>>;
+}
```

Comment:
What is a purpose of this trait?

Reply:
Helper functions to provide common functionalities which will be used in many tests